CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 2.4.0
+
+* [PCP-862](https://tickets.puppetlabs.com/browse/PCP-862) Add disconnect function
+  This adds a disconnect function to disconnect PCP connections rather than
```

Comment:
We probably don't need all this context, but I guess it doesn't hurt.

Reply:
Actually, I don't think we should mention PCP here. I'd just make this a new feature to allow disconnecting websocket connections, it's not specific to PCP.